pkg/crypto/fips_runtime_drift_test.go

```
@@ -0,0 +1,135 @@
+// whether the handshake completes in the current process's FIPS mode.
+func canNegotiateGroup(t *testing.T, cert tls.Certificate, curve tls.CurveID) error {
+	t.Helper()
+	ln, err := tls.Listen("tcp", "127.0.0.1:0", &tls.Config{
+		Certificates:     []tls.Certificate{cert},
+		MinVersion:       tls.VersionTLS13,
+		CurvePreferences: []tls.CurveID{curve},
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer ln.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `ln.Close` is not checked

(errcheck)